sunpy/map/sources/sdo.py

```diff
@@ -43,15 +42,23 @@ def processing_level(self):
 
     def _get_mpl_normalizer(self):
         """Returns a Normalize object to be used with AIA data"""
-        # byte-scaled images have most likely already been scaled
-        if self.data.dtype == np.uint8:
-            return None
-
-        mean = self.mean()
-        std = self.std()
-
-        vmin = max(0, mean - 3 * std)
-        vmax = min(self.max(), mean + 3 * std)
+        vmin = self.data.min()
+        vmax = self.data.max()
+
+        if self.wavelength.value == 304:
```

Comment:
If it was a dict then we could provide it as a user-default so people can choose their own normalizations at start-up...but I don't think I'm going to do that in this PR :P